client/components/frontpage/HeroSection.js

```diff
@@ -0,0 +1,116 @@
+import React from "react";
+import Link from "next/link";
+import styled, { css } from "styled-components";
+import { breakpoint } from "../../frontend-config";
+
+const Wrapper = styled.div`
+  background: #333;
+  padding: 45px 25px 65px;
+
+  @media (min-width: ${breakpoint("lg")}) {
+    padding: 115px 25px 190px;
+  }
+`;
+
+const ContentWrapper = styled.div`
+  max-width: 730px;
+  margin: 0 auto;
+`;
+
+const Headline = styled.h1`
+  font-size: 25px;
+  line-height: 30px;
+  font-weight: 700;
+  letter-spacing: 0.5px;
+  text-align: center;
+  color: #18e28c;
+  margin-bottom: 32px;
+
+  @media (min-width: ${breakpoint("md")}) {
+    font-size: 40px;
+    line-height: 44px;
+  }
+
+  @media (min-width: ${breakpoint("lg")}) {
+    font-size: 53px;
+    line-height: 63px;
+  }
+`;
+
+const Subtitle = styled.p`
+  font-size: 16px;
+  line-height: 24px;
+  text-align: center;
+  color: #fff;
+  padding: 0 40px;
+
+  @media (min-width: ${breakpoint("lg")}) {
+    font-size: 18px;
+    line-height: 28px;
+    padding: 0;
+  }
+`;
+
+const ButtonsWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-top: 40px;
+`;
+
+const Button = styled.button`
+  background: transparent;
```

Comment:
@ZumDeWald This is almost an exact duplicate of your file. I'll probably change this around once your code is through to develop